roles/remote/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+user: hitchwiki
+hostname: hitchwiki.test
```

Comment:
what does this hostname refer to? Note that it's on config file which is pushed to repo, so it's kind of not configurable.

Reply:
it was no longer used, dropped the file